frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -337,7 +336,7 @@
           <CrdtSyncButton {project} {isEmpty} />
         </FeatureFlagContent>
       {/if}
-      {#if project.type === ProjectType.FlEx && $isDev && !isEmpty}
+      {#if project.type === ProjectType.FlEx && !isEmpty}
```

Comment:
Playwright tests are broken, because our `if` logic doesn't make sense.

I think it should be:
`if (canAskToJoinProject)`
`else if (project.type === ProjectType.FlEx && !isEmpty)`
`else ...`

Reply:
Yes, I was quite new to the Playwright tests, but that one that's failing looks pretty janky. Thanks for the recommendation!